package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fusion-plugin-browser-performance-emitter",
   "decsription": "Emit performance stats from the browser",
-  "version": "1.4.3",
+  "version": "1.5.0",
```

Comment:
This should be a separate PR after, and I guess we're doing pre-releases now instead of releases...

Reply:
We're already on 1.5.0 (https://github.com/fusionjs/fusion-plugin-browser-performance-emitter/releases). Unclear why the diff shows 1.4.3

Reply:
This is fine then, ignore me. Thanks!